source/modules/configuration/external_links.rst

```diff
@@ -4,50 +4,111 @@ External links
 Some elements of GLPI can be associated with a set of links to external applications.
 These are visible from the **Links** tab of the various forms.
 
-To parameterize a link, it is possible to use tags which will be replaced by the values of the element.
-The tags are:
-
-- [LOGIN]: The logged in user's username
-- [ID]: The internal numeric ID for the item
-- [NAME]: The name of the item
-- [LOCATION]: The name of the item's location
-- [LOCATIONID]: The internal numeric ID for the item's location
-- [IP]: The IP address of the item
-- [MAC]: The MAC address of the item
-- [NETWORK]: The item's network
-- [DOMAIN]: The item's domain. If more than one domain is associated with the item, the first one is used.
-- [SERIAL]: The item's serial number
-- [OTHERSERIAL]: The item's inventory number/asset tag
-- [USER]: The item's user
-- [GROUP]: The item's group
-- [FIRSTNAME]: The user's first name (Only applies to User external links)
-- [REALNAME]: The user's surname (Only applies to User external links)
-- [FIELD:*]: If the field you want is not available as a separate tag, it may still be possible to use it by referencing its internal field name.
-    For example, the comment field would be `[FIELD:comment]`.
-    The field name is case sensitive and typically it will need to be all lowercase.
+Add an external link
+^^^^^^^^^^^^^^^^^^^^
+
+* Click on **+ Add**
+* Enter a **name**
+* Enter an **URL**
+* Choose your **icon**
+* You can choose to open it in a **different tab**
+
+
+.. image:: ../tabs/images/external_link_add.png
+    :alt: Add external link
+    :scale: 43%
+
+From a GLPI object, select the newly created link so that it appears in the Links tab.
+
+.. image:: ../tabs/images/external_link_view.png
+    :alt: View external link
+    :scale: 48%
+
+
+
+Add an "generic" external link
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can add a generic link for all elements of a GLPI object.
+
+* In the **links** tab of your objet, click on **Configure XXXXXXXX links** (or go to **Setup** > **External Links**)
+* Click on **+ Add**
+
+To setup a link, it is possible to use tags which will be replaced by the values of the element
+
+The tags are :
```

Comment:
External links now use Twig templates. Existing links will be converted automatically during the upgrade to GLPI 11, but new links need to use Twig syntax. [NAME] -> {{ NAME }}.

https://github.com/glpi-project/glpi/pull/16502

Reply:
I'm correcting and adding your comment, which provides good information